multipaz/src/commonMain/kotlin/org/multipaz/util/Logger.kt

```diff
@@ -36,14 +36,19 @@ import kotlinx.serialization.json.JsonElement
 object Logger {
     private const val TAG = "Logger"
 
-    internal enum class Level {
+    enum class Level {
         DEBUG,
         INFO,
         WARNING,
         ERROR,
     }
 
+    fun interface LogPrinter {
+        fun print(level: Level, tag: String, msg: String, throwable: Throwable?)
+    }
+
     var isDebugEnabled = true // TODO: make false by default
+    var logPrinter: LogPrinter? = null
```

Comment:
Please add docs for  `Level`, `LogPrinter` types and `logPrinter` instance.

Reply:
Ah, right. Fixed in 7682636aeabeae28966748cacd244541fdea3b51. Realized also that `Level` probably should be moved under `LogPrinter`.